build/Targets/ProducesNoOutput.Settings.targets

```diff
@@ -3,20 +3,13 @@
 
   <PropertyGroup>
     <GenerateDependencyFile>false</GenerateDependencyFile>
-    <IsDeployment Condition="'$(IsDeployment)' == ''">false</IsDeployment>
   </PropertyGroup>
 
-  <PropertyGroup Condition="'$(IsDeployment)' == 'false'">
```

Comment:
is `IsDeployment` used just to set `UseCommonOutputDirectory` and eventually set the `OutDir` as well?